shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -110,6 +111,7 @@ class DiffTests extends org.scalatest.funsuite.AnyFunSuite with org.scalatest.Pa
           case "ng" => mode.copy(noGeneration = true)
           case "js" => mode.copy(showGeneratedJS = true)
           case "ts" => mode.copy(generateTsDeclarations = true)
+          case "variance" => mode.copy(showVariance = true)
```

Comment:
I think it would be more useful to have a `dv` flag (for "debug variance") which shows the fixed analysis point progressing through the use of `trace`.